README.md

```diff
@@ -52,6 +52,38 @@ Please note you have to set the environment variables as described below and mou
+| DD_PRODUCT_TYPE_NAME  | Mandatory          | Mandatory        | A product type with this name must exist |
+| DD_PRODUCT_NAME       | Mandatory          | Mandatory        | If a product with this name does not exist, it will be created |
+| DD_ENGAGEMENT_NAME    | Mandatory          | -                | If an engagement with this name does not exist for the given product, it will be created |
+| DD_ENGAGEMENT_TARGET_START | Optional    | -                | Format: YYYY-MM-DD, default: `today`. The target start date for a newly created engagement. |
+| DD_ENGAGEMENT_TARGET_END | Optional    | -                | Format: YYYY-MM-DD, default: `2999-12-31`. The target start date for a newly created engagement. |
+| DD_TEST_NAME          | Mandatory          | -                | If a test with this name does not exist for the given engagement, it will be created |
+| DD_TEST_TYPE_NAME     | Mandatory          | -                | From DefectDojo's list of test types, eg. `Trivy Scan` |
+| DD_FILE_NAME          | Optional           | Mandatory        | |
+| DD_ACTIVE             | Optional           | -                | Default: `true` |
+| DD_VERIFIED           | Optional           | -                | Default: `true` |
+| DD_MINIMUM_SEVERITY   | Optional           | -                | |
+| DD_GROUP_BY           | Optional           | -                | Group by file path, component name, component name + version |
+| DD_PUSH_TO_JIRA       | Optional           | -                | Default: `false` |
+| DD_CLOSE_OLD_FINDINGS | Optional           | -                | Default: `true` |
+| DD_VERSION            | Optional           | -                | |
+| DD_ENDPOINT_ID        | Optional           | -                | |
+| DD_SERVICE            | Optional           | -                | |
+| DD_BUILD_ID           | Optional           | -                | |
+| DD_COMMIT_HASH        | Optional           | -                | |
+| DD_BRANCH_TAG         | Optional           | -                | |
+| DD_API_SCAN_CONFIGURATION_ID | Optional    | -                | Id of the API scan configuration for API based parsers, e.g. SonarQube |
+| DD_SSL_VERIFY         | Optional           | Optional         | Disable SSL verification by setting to `false` or `0`. Default: `true` |
+| DD_EXTRA_HEADER_1         | Optional           | Optional         | If extra header key is needed for auth in wafs or similar |
+| DD_EXTRA_HEADER_1_VALUE   | Optional           | Optional         | The corresponding value for extra header key |
+| DD_EXTRA_HEADER_2         | Optional           | Optional         | If extra header key is needed for auth in wafs or similar |
+| DD_EXTRA_HEADER_2_VALUE   | Optional           | Optional         | The corresponding value for extra header key |
+=======
 | Parameter                           | Re-import findings | Import languages | Remark                                                                                            |
```

Comment:
Here the table starts again. The conflict has not been removed properly, now there are the old and the new version of the table in the file.